flink-doris-connector/src/main/java/org/apache/doris/flink/deserialization/DorisJsonDebeziumDeserializationSchema.java

```diff
@@ -111,65 +116,74 @@ private JsonNode convertToJson(Schema schema, Object value) throws DorisExceptio
+                            Schema valueSchema = schema == null ? null : schema.valueSchema();
+                            JsonNode fieldValue = convertToJson(valueSchema, elem);
+                            list.add(fieldValue);
+                        }
+                        return list;
                     }
-                    return list;
-                }
-                case MAP: {
-                    Map<?, ?> map = (Map<?, ?>) value;
-                    // If true, using string keys and JSON object; if false, using non-string keys and Array-encoding
-                    boolean objectMode;
-                    if (schema == null) {
-                        objectMode = true;
-                        for (Map.Entry<?, ?> entry : map.entrySet()) {
-                            if (!(entry.getKey() instanceof String)) {
-                                objectMode = false;
-                                break;
+                case MAP:
+                    {
+                        Map<?, ?> map = (Map<?, ?>) value;
+                        // If true, using string keys and JSON object; if false, using non-string
+                        // keys and Array-encoding
+                        boolean objectMode;
+                        if (schema == null) {
+                            objectMode = true;
+                            for (Map.Entry<?, ?> entry : map.entrySet()) {
+                                if (!(entry.getKey() instanceof String)) {
+                                    objectMode = false;
+                                    break;
+                                }
                             }
+                        } else {
+                            objectMode = schema.keySchema().type() == Schema.Type.STRING;
                         }
-                    } else {
-                        objectMode = schema.keySchema().type() == Schema.Type.STRING;
-                    }
-                    ObjectNode obj = null;
-                    ArrayNode list = null;
-                    if (objectMode) {
-                        obj = JSON_NODE_FACTORY.objectNode();
-                    } else {
-                        list = JSON_NODE_FACTORY.arrayNode();
-                    }
-                    for (Map.Entry<?, ?> entry : map.entrySet()) {
-                        Schema keySchema = schema == null ? null : schema.keySchema();
-                        Schema valueSchema = schema == null ? null : schema.valueSchema();
-                        JsonNode mapKey = convertToJson(keySchema, entry.getKey());
-                        JsonNode mapValue = convertToJson(valueSchema, entry.getValue());
-
+                        ObjectNode obj = null;
+                        ArrayNode list = null;
                         if (objectMode) {
-                            obj.set(mapKey.asText(), mapValue);
+                            obj = JSON_NODE_FACTORY.objectNode();
```

Comment:
Is this as expected?

Reply:
As expected, due to the previous code formatting, the code corresponding to line `155` is on line `164`.

Reply:
Due to code formatting reasons, the actual line 153 corresponds to line 164.